Gemfile

```diff
@@ -2,19 +2,19 @@ source 'https://rubygems.org'
 
 ruby File.read('.ruby-version').chomp
 
-gem 'actionpack', '~> 6'
-gem 'actionview', '~> 6'
-gem 'activemodel', '~> 6'
-gem 'activerecord', '~> 6'
-gem 'activesupport', '~> 6'
+gem 'actionpack'
+gem 'actionview'
+gem 'activemodel'
+gem 'activerecord'
+gem 'activesupport'
```

Comment:
I guess we should specify 
`, '~> 6'`
in here.

Reply:
Not sure :shrug:

Reply:
Given we have a lock file and updates are very much under our control I don't see the advantage

Reply:
Spoken about this - basically it reduces headaches where we have to manage updates that push us to rails 8 before we're ready